src/blocks/HelpfulFeedback/style.scss

```
@@ -0,0 +1,294 @@
+        .wedocs-feedback-buttons {
+            display: none;
+        }
+    }
+
+    // Animation for feedback transitions
+    .wedocs-feedback-thank-you,
+    .wedocs-feedback-buttons {
+        animation: fadeIn 0.3s ease-in-out;
+    }
+
+    @keyframes fadeIn {
```

Stylelint (17.11.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)